pupetteer.helpers.js

```diff
@@ -26,6 +26,6 @@ module.exports = {
 	},
 	clickElementBySelector: async (page, selector, text) => {
 		const element = await getElementWithSelector(page, selector, text)
-		return await page.click(element)
+		return await element.click()
```

Comment:
@sercanvirlan should this have to be page.click: https://pptr.dev/api/puppeteer.page.click/

Reply:
This was not working with page.click, element click is working.

Reply:
Hmm. Maybe puppeteer apis are updated. Ok if it's working then great. Can you add a feature that uses the click step function so this codebase would be fully working? I did write the google search with click first but then google had autocompelte that keypress-enter was the suggested method so this step function got unused.